docs/pages/machine-id/getting-started.mdx

```diff
@@ -4,24 +4,25 @@ description: Getting started with Teleport Machine ID
 ---
 
 In this getting started guide, you will use Machine ID to create a bot user for
-a machine and use that identity to connect to said machine.
+a machine, then use that identity to connect to the machine.
```

Comment:
I think this is technically inaccurate.

The machine you are connecting to is a "host". The identity machine ID is issuing is for a [bot] "user" to connect to said host.

In other words, the identity you create and the machine you connect to are separate. This wording makes them sound the same.